smriprep/workflows/anatomical.py

```diff
@@ -22,15 +25,17 @@
 )
 from niworkflows.interfaces.images import TemplateDimensions, Conform, ValidateImage
 from niworkflows.utils.misc import fix_multi_T1w_source_name, add_suffix
-from niworkflows.anat.ants import init_brain_extraction_wf
+from niworkflows.anat.ants import (
+    init_brain_extraction_wf, init_n4_only_wf, _pop
```

Comment:
not a huge fan of importing "private" functions

Reply:
Sure, it's just a pop function so we can redefine.